release/internal/registry/component.go

```diff
@@ -0,0 +1,36 @@
+package registry
```

Comment:
We need copyright headers on all of our files

Reply:
I'd probably just put this in pinnedversion.go, though - it's just a substruct of the PinnedVersion structure.